### PR TITLE
newdevice: Lancom wireless devices

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Fixes: #4789 - cobbled this together from the issue :/

### CODE REVIEW

File: includes/definitions/lcos.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+os: lcos
+text: 'Lancom OS'
+type: firewall
```
Comment (Rosiak) [inline includes/definitions/lcos.yaml]: I belive this is a typo :)

Comment (laf) [inline includes/definitions/lcos.yaml]: Which bit?

Comment (Rosiak) [inline includes/definitions/lcos.yaml]: Firewall, shouldn't it be wireless?

Comment (laf) [inline includes/definitions/lcos.yaml]: Derp :( Spotted, fixed and pushed

**Pull request merged**